src/providers/image-loader.ts

```diff
@@ -414,12 +414,19 @@ export class ImageLoader {
    * @returns {Promise<string>} Returns a promise that resolves with the local path if exists, or rejects if doesn't exist
    */
   private getCachedImagePath(url: string): Promise<string> {
+    // Check if we're running with livereload
+    let _isDev: boolean = (window['IonicDevServer'] != undefined);
```

Comment:
1. If the `IonicDevServer` variable is defined before the app loads, it makes more sense to make this a global variable (either a variable outside the class, or a private property of the class). This way we don't have to evaluate the expression every time `getCachedImagePath` is called.